libs::ncurses-dev build fails

```
DESTDIR=/home/superq/ws/new/dev/build/libs/ncurses-dev/x86_64-wendy-linux-gnu/1/workspace/build/../install \
prefix=/usr \
exec_prefix=/usr \
bindir=/usr/bin \
top_srcdir=/home/superq/ws/new/dev/src/libs/ncurses/1/workspace \
srcdir=/home/superq/ws/new/dev/src/libs/ncurses/1/workspace/misc \
datadir=/usr/share \
ticdir=/usr/share/terminfo \
source=terminfo.tmp \
cross_compiling=yes \
/bin/sh ./run_tic.sh
** Building terminfo database, please wait...
Running tic to install /home/superq/ws/new/dev/build/libs/ncurses-dev/x86_64-wendy-linux-gnu/1/workspace/build/../install/usr/share/terminfo ...

        You may see messages regarding extended capabilities, e.g., AX.
        These are extended terminal capabilities which are compiled
        using
                tic -x
        If you have ncurses 4.2 applications, you should read the INSTALL
        document, and install the terminfo without the -x option.

"terminfo.tmp", line 4790, terminal 'xterm-16color': error writing /home/superq/ws/new/dev/build/libs/ncurses-dev/x86_64-wendy-linux-gnu/1/workspace/install/usr/share/terminfo/x/xterm-16color
? tic could not build /home/superq/ws/new/dev/build/libs/ncurses-dev/x86_64-wendy-linux-gnu/1/workspace/build/../install/usr/share/terminfo
Makefile:93: recipe for target 'install.data' failed
make[1]: Leaving directory '/home/superq/ws/new/dev/build/libs/ncurses-dev/x86_64-wendy-linux-gnu/1/workspace/build/misc'
make[1]: *** [install.data] Error 1
Makefile:118: recipe for target 'install' failed
make: *** [install] Error 2
++ bob_handle_error 2
++ set +x
Step failed with return status 2; Command: make $INSTALL_TAGET DESTDIR=${PWD}/../install ${INSTALL_OPTIONS:+"${INSTALL_OPTIONS[@]}"}
Call stack (most recent call first)
    #0: Class  autotools, layer basement, line 62, in autotoolsBuild
    #1: Recipe libs::ncurses#1, layer basement, line 1, in main
```

--- a/recipes/libs/ncurses.yaml
+++ b/recipes/libs/ncurses.yaml
@@ -11,6 +11,17 @@ checkoutSCM:
 
 buildTools: [host-toolchain]
 buildScript: |
+    # We need to pass the path of the just built tic
+    # able to run on the building machine, so that the
+    # terminal database can be created without errors.
+    mkdir tic
+    pushd tic
+    $1/configure
+    make -C include
+    make -C progs tic
+    export TIC_PATH=$(pwd)/progs/tic
+    popd
+
     autotoolsBuild $1 \
             --with-shared           \
             --without-debug         \

